package.json

```diff
@@ -5,9 +5,9 @@
   "description": "Ferdium server to replace the default Franz/Ferdi server.",
   "main": "index.js",
   "engines": {
-    "node": "^14.17",
-    "npm": "^6.14",
-    "node-gyp": "^8.0"
+    "node": "16.14.2",
+    "npm": "8.7.0",
+    "node-gyp": "9.0.0"
```

Comment:
Here you are referring to `node-gyp` v9.0.0, but in the `CONTRIBUTING.md`, its 8.7.0. Also, if you are anyways installing if as an explicit dependency here, then why include it as a system pre-requisite? Please delete from that file - it can get outdated.

Reply:
Okay will remove `node-gyp` from `CONTRIBUTING.md`